aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -512,66 +521,54 @@ def export(
 
     writer = get_writer(file_format)(filename=filename, use_compression=use_compression, **(writer_init or {}))
 
-    if silent:
-        logging.disable(logging.CRITICAL)
-
-    try:
-        summary(
-            file_format=writer.file_format_verbose,
-            export_version=writer.export_version,
-            outfile=filename,
-            include_comments=include_comments,
-            include_logs=include_logs,
-            traversal_rules=full_traversal_rules
-        )
+    summary(
+        file_format=writer.file_format_verbose,
+        export_version=writer.export_version,
+        outfile=filename,
+        include_comments=include_comments,
+        include_logs=include_logs,
+        traversal_rules=full_traversal_rules
+    )
 
-        report_data: Dict[str, Any] = {'time_write_start': None, 'time_write_stop': None, 'writer_data': None}
-
-        report_data['time_collect_start'] = time.time()
-        export_data = _collect_archive_data(
-            entities=entities,
-            silent=silent,
-            allowed_licenses=allowed_licenses,
-            forbidden_licenses=forbidden_licenses,
-            include_comments=include_comments,
-            include_logs=include_logs,
-            **traversal_rules
-        )
-        report_data['time_collect_stop'] = time.time()
+    report_data: Dict[str, Any] = {'time_write_start': None, 'time_write_stop': None, 'writer_data': None}
 
-        extract_time = report_data['time_collect_stop'] - report_data['time_collect_start']
-        EXPORT_LOGGER.debug(f'Data extracted in {extract_time:6.2g} s.')
+    report_data['time_collect_start'] = time.time()
+    export_data = _collect_archive_data(
+        entities=entities,
+        allowed_licenses=allowed_licenses,
+        forbidden_licenses=forbidden_licenses,
+        include_comments=include_comments,
+        include_logs=include_logs,
+        **traversal_rules
+    )
+    report_data['time_collect_stop'] = time.time()
 
-        if export_data is not None:
-            try:
-                report_data['time_write_start'] = time.time()
-                report_data['writer_data'] = writer.write(export_data=export_data, silent=silent)  # type: ignore
-                report_data['time_write_stop'] = time.time()
-            except (exceptions.ArchiveExportError, LicensingException) as exc:
-                if os.path.exists(filename):
-                    os.remove(filename)
-                raise exc
+    extract_time = report_data['time_collect_stop'] - report_data['time_collect_start']
+    EXPORT_LOGGER.debug(f'Data extracted in {extract_time:6.2g} s.')
 
-            write_time = report_data['time_write_stop'] - report_data['time_write_start']
-            EXPORT_LOGGER.debug(f'Data written in {write_time:6.2g} s.')
+    if export_data is None:
```

Comment:
```suggestion
    if export_data is not None:
```

fix accidental change (whilst testing)